source/cellular_at_core.c

```diff
@@ -43,10 +43,14 @@
 
 /*-----------------------------------------------------------*/
 
-#define CHECK_IS_PREFIX_CHAR( inputChar )                                 \
+#if ( CELLULAR_CONFIG_MODULE_CHECK_PFREFIX_CHAR == 0 )
+    #define CHECK_IS_PREFIX_CHAR( inputChar )                             \
```

Comment:
Would adding a #ifndef be enough which will give a chance to override it in the config file? 

```
#ifndef CHECK_IS_PREFIX_CHAR
#define CHECK_IS_PREFIX_CHAR( inputChar ) \
( ( ( ( int32_t ) isalpha( ( ( int8_t ) ( inputChar ) ) ) ) == 0 ) && \
      ( ( ( int32_t ) isdigit( ( ( int8_t ) ( inputChar ) ) ) ) == 0 ) && \
      ( ( inputChar ) != '+' ) && ( ( inputChar ) != '_' ) )
#endif
```

Reply:
Update in this PR.